Tasks/IISWebAppDeploymentOnMachineGroup/Tests/L0.ts

```diff
@@ -1,15 +1,23 @@
 import * as path from 'path';
 import * as assert from 'assert';
 import * as ttm from 'vsts-task-lib/mock-test';
+import tl = require('vsts-task-lib');
+var ltx = require('ltx');
+import fs = require('fs');
 
 describe('IISWebsiteDeploymentOnMachineGroup test suite', function() {
      var taskSrcPath = path.join(__dirname, '..','deployiiswebapp.js');
 
      before((done) => {
+        tl.cp(path.join(__dirname, 'L1XmlVarSub/Web.config'), path.join(__dirname, 'L1XmlVarSub/Web_test.config'), null, true);
+        tl.cp(path.join(__dirname, 'L1XmlVarSub/Web.Debug.config'), path.join(__dirname, 'L1XmlVarSub/Web_test.Debug.config'), null, true);
```

Comment:
These tests I believe are getting duplicated across Azure App Service Task and Machine Group Task.  Can we write tests for utility functions inside common folder itself.